### PR TITLE
fix typo in `torch.sum` documentation

### PR DESCRIPTION
**Pull request by umanwizard**
Notice that an extra colon was added to `:attr:`, so in https://pytorch.org/docs/stable/torch.html#torch.sum , `dim` shows up as ":attr::_dim_". This patch fixes the issue.

### CODE REVIEW

**Review by soumith — APPROVED** [review]

**Comment by soumith** [comment]
thank you @umanwizard !
